core/src/main/scala/akka/persistence/cassandra/query/TagViewSequenceNumberScanner.scala

```diff
@@ -109,14 +109,13 @@ import akka.stream.alpakka.cassandra.scaladsl.CassandraSession
             }
             acc + (pid -> ((newTagPidSequenceNr, newTimestamp)))
         }
-        .flatMap { result =>
-          if (deadline.hasTimeLeft()) {
-            doIt()
```

Comment:
@chbatey What do you think about this? As far as I can see it's just throwing away earlier results.

Reply:
Looks good

Reply:
I think the initial thought was the scan would take some time so the timeout would have met by the first time it came back